scripts/ci/upload_artifacts.sh

```diff
@@ -9,16 +9,25 @@ then
 
   if [[ "${CI_TAG}" ]];
   then
-    FILENAME="qfield-${CI_TAG}-${ARCH}.apk"
+    FILENAME_AAB="qfield-${CI_TAG}-${ARCH}.aab"
+    FILENAME_APK="qfield-${CI_TAG}-${ARCH}.apk"
   else
-    FILENAME="qfield-dev-${CI_UPLOAD_ARTIFACT_ID}-${CI_COMMIT}-${ARCH}.apk"
+    FILENAME_AAB="qfield-dev-${CI_UPLOAD_ARTIFACT_ID}-${CI_COMMIT}-${ARCH}.aab"
+    FILENAME_APK="qfield-dev-${CI_UPLOAD_ARTIFACT_ID}-${CI_COMMIT}-${ARCH}.apk"
   fi
 
-  mv ${CMAKE_BUILD_DIR}/android-build/build/outputs/apk/release/android-build-release-signed.apk /tmp/${FILENAME}
```

Comment:
Do we need to preserve this line? Something is still done with it later

Reply:
@m-kuhn , it's there, a few lines below (line 26)

Reply:
Ah, missed that